javasphinx/util.py

```diff
@@ -14,13 +14,15 @@
 # limitations under the License.
 #
 
+from __future__ import unicode_literals
+
```

Comment:
Lets add `from builtins import str` here so this is consistently unicode between Python 2.x and 3.x